eclair-core/src/main/scala/fr/acinq/eclair/io/Server.scala

```diff
@@ -55,7 +55,6 @@ class Server(nodeParams: NodeParams, switchboard: ActorRef, router: ActorRef, ad
   def listening(listener: ActorRef): Receive = {
     case Connected(remote, _) =>
       log.info(s"connected to $remote")
-      Metrics.PeerConnections.withTag(Tags.ConnectionState, Tags.ConnectionStates.Connected).increment()
```

Comment:
Good catch!

Reply:
nit: the `import` director for Metrics can be removed now